recipes/pdfminer/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{%set name = "pdfminer" %}
+{%set version = "20140328" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "ba187b93056586a14edd5e630cf63ae96b1cc84b611f55dcddbc997316d9f262" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - pdfminer
+
+about:
+  home: http://euske.github.io/pdfminer/index.html
+  license: MIT
```

Comment:
Unfortunately they don't seem to include a license file. Please see issue ( https://github.com/euske/pdfminer/issues/142 ) for a request to add this. We will have to address this once they provide one to add.